.github/workflows/test-integration-vizro-ai.yml

```
@@ -30,7 +30,9 @@ jobs:
             hatch-env: all.py3.10
           - python-version: "3.11"
             hatch-env: all.py3.11
-          - python-version: "3.11"
+          - python-version: "3.12"
+            hatch-env: all.py3.12
+          - python-version: "3.12"
```

```suggestion
          - python-version: "3.11"
```

Given Python 3.12 is pretty new and a bit problematic, let's keep 3.11 as our "standard" version for running stuff in CI, including as the version for `lower-bounds`.

Same change would apply across all the github workflows.